Database/compiler.lua

```diff
@@ -715,30 +715,30 @@ function QuestieDBCompiler:Compile(finalize)
     end
     QuestieDBCompiler.startTime = GetTime()
     QuestieDBCompiler.totalSize = 0
-    print(QuestieLocale:GetUIString("\124cFF4DDBFF [1/4] Updating NPCs..."))
+    print(l10n("\124cFF4DDBFF [1/4] Updating NPCs..."))
```

Comment:
These messages should be just "Updating NPCs..." and the rest hardcoded + `Questie:Colorize`